.github/workflows/ci.yaml

```diff
@@ -18,19 +18,26 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
+
       - name: Reject trailing whitespace
         run: ./utils/trailing-whitespace.sh reject
-      - name: what's my env
+
+      - name: What's my env
         run:  env
-      - name: whoami
+
+      - name: Whoami
         run:  whoami
-      - name: which rustup
+
+      - name: Which rustup
         run:  which rustup
-      - name: rustup show
+
+      - name: Rustup show
         run:  rustup show
+
       - name: Check formatting of code
         run: cargo +stable fmt --check
-      - name: Clippy Checks
+
+      - name: Clippy checks
         run: cargo +stable clippy
 
   test:
```

Comment:
name: Test
can this be more specific?

Reply:
hmm what do you suggest? unit-and-integration-test maybe?

Reply:
or cargo-test ?

Reply:
all our test jobs are not plural so `Cargo test` fits the format and also is the actual command so i think thats better